### PR TITLE
[RedTube] fixed redtube test case md5,upload date and duration

### PR DESCRIPTION
**Pull request by Wandang**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Fixes test case for redtube. Updated md5, upload date and duration.

### CODE REVIEW

File: youtube_dl/extractor/redtube.py [hunk 1]
```diff
@@ -16,13 +16,13 @@ class RedTubeIE(InfoExtractor):
...
-        'md5': '7b8c22b5e7098a3e1c09709df1126d2d',
+        'md5': 'fc08071233725f26b8f014dba9590005',
         'info_dict': {
             'id': '66418',
             'ext': 'mp4',
             'title': 'Sucked on a toilet',
-            'upload_date': '20120831',
-            'duration': 596,
+            'upload_date': '20110811',
+            'duration': None,
```
Comment (dstftw) [inline youtube_dl/extractor/redtube.py]: Fix duration extraction instead.

**Comment by Wandang** [comment]
Done, hope this code is ok, still learning

File: youtube_dl/extractor/redtube.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def _real_extract(self, url):
             r'<span[^>]+>ADDED ([^<]+)<',
             webpage, 'upload date', fatal=False))
         duration = int_or_none(self._search_regex(
-            r'videoDuration\s*:\s*(\d+)', webpage, 'duration', default=None))
+            '<meta.*property="og:video:duration".*content="(.*)".*\/>', webpage, 'duration', default=None))
```
Comment (dstftw) [inline youtube_dl/extractor/redtube.py]: _og_search_property`.

File: youtube_dl/extractor/redtube.py [hunk 3]
```diff
@@ -91,7 +91,7 @@ def _real_extract(self, url):
             r'<span[^>]+>ADDED ([^<]+)<',
             webpage, 'upload date', fatal=False))
         duration = int_or_none(self._search_regex(
-            r'videoDuration\s*:\s*(\d+)', webpage, 'duration', default=None))
```
Comment (dstftw) [inline youtube_dl/extractor/redtube.py]: Keep the old fallback also.

File: youtube_dl/extractor/redtube.py (hunk 2 above)
Comment (Wandang) [reply youtube_dl/extractor/redtube.py]: thx for your commitment to help out a newbie. I will read the code guidelines fully before I continue

**Comment by Wandang** [comment]
I updated my code (Do I need to write a message/comment everytime or is this noticeable via github?)

**Pull request merged**